eth_worker/_docker_worker_script.sh

```diff
@@ -29,7 +29,7 @@ if [ "$CONTAINER_MODE" == 'ETH_WORKER_TEST' ]; then
 else
     if [ "$CONTAINER_TYPE" == 'BEAT' ]; then
```

Comment:
we need to add a beat worker to dockerrun.aws.json I think.... (and docker compose too maybe?)